pants.toml

```diff
@@ -14,10 +14,3 @@ repo_id = "205E301D-44A4-42FF-8495-C9D41871A431"
 
 [python]
 interpreter_constraints = ["CPython>=3.8,<3.11"]
-enable_resolves = true
-
-[python-protobuf]
-runtime_dependencies = ["3rdparty/python#protobuf"]
```

Comment:
Woohoo!

Reply:
cc @stuhood re your "code golf" comments from Podcast.__init__! We're now at 11 lines for this pants.toml, 3 of which is anonymous telemtry.